icepyx/tests/test_auth.py

```diff
@@ -18,15 +18,15 @@ def auth_instance():
 
 
 # Test that .session creates a session
-@pytest.mark.skipif(os.getenv("TRAVIS"), "Skipping this test on Travis CI.")
+@pytest.mark.skipif(os.getenv("TRAVIS"), reason="Skipping this test on Travis CI.")
 def test_get_session(auth_instance):
     print(os.environ)
     assert isinstance(auth_instance.session, requests.sessions.Session)
 
 
 # Test that .s3login_credentials creates a dict with the correct keys
 @pytest.mark.skipif(
-    os.environ.get("TRAVIS") == "true", "Skipping this test on Travis CI."
+    os.environ.get("TRAVIS") == "true", reason="Skipping this test on Travis CI."
```

Comment:
Nitpick: Let's use either `os.environ.get` or `os.getenv` in the same way for each mark.

Reply:
👍 Once I get it to work, I will do it the same way in all three places.

Reply:
Sorry if I'm getting ahead of things here :)